acceptance/tests/inventory/basic_fact_retrieval.rb

```diff
@@ -40,10 +40,13 @@
     end
 
     step "create a custom structured fact" do
+      time = Time.now.iso8601
       payload = <<-EOM
       -H "Accept: application/json" -H "Content-Type: application/json" \
       -d '{"command":"replace facts","version":4, \
       "payload":{"environment":"DEV","certname":"#{master}", \
+      "timestamp": "#{time}"\
+      "producer_timestamp": "#{time}"\
```

Comment:
@mullr there's a missing comma after timestamp here -- I think that's contributing to your woe :)